src/services/api/features/package-actions/handlers/issue-rai-lambda.ts

```diff
@@ -0,0 +1,37 @@
+import { type APIGatewayEvent } from "aws-lambda";
+import { handleEvent } from "../lambda-handler";
+import { response } from "@/libs/handler";
+import { PackageActionWriteService } from "../services/package-action-write-service";
+import { seatoolConnection } from "../consts/sql-connection";
+import { kafkaConfig } from "../consts/kafka-connection";
+import { raiIssueSchema } from "shared-types";
+import { TOPIC_NAME } from "../consts/kafka-topic-name";
+
+export const issueRaiLambda = async (event: APIGatewayEvent) => {
+  return await handleEvent({
+    event,
+    schema: raiIssueSchema, // don't use this
+    async lambda(data) {
+      const packageActionService =
+        await PackageActionWriteService.createPackageActionWriteService(
+          kafkaConfig,
+          seatoolConnection,
+          TOPIC_NAME
+        );
+
+      await packageActionService.issueRai(data);
+
+      await packageActionService.changePackageStatus({
+        id: data.id,
+        status: "Pending-RAI",
+      });
+
+      return response({
+        statusCode: 200,
+        body: {
+          message: "rai issued successfully",
```

Comment:
That is the plan. After discusssions with Paul. The thought is that these -lambda files return simply a javascript object and then at the router level it will return the response there and only there. Thoughts on that?